ui/src/scss/_utilities.scss

```diff
@@ -143,4 +143,9 @@
   .u-width--full {
     width: 100%;
   }
+
+  .p-text--default-size {
```

Comment:
`.u-text--default-size` if it's in utilities?

Reply:
Thanks.